src/main/java/org/fcrepo/camel/processor/SparqlUpdateProcessor.java

```diff
@@ -44,43 +52,24 @@
      *
      * @param exchange the current camel message exchange
      */
-    public void process(final Exchange exchange) throws IOException {
+    public void process(final Exchange exchange) throws IOException, NoSuchHeaderException {
 
         final Message in = exchange.getIn();
 
         final ByteArrayOutputStream serializedGraph = new ByteArrayOutputStream();
-        final String subject = ProcessorUtils.getSubjectUri(in);
-        final String namedGraph = in.getHeader(FcrepoHeaders.FCREPO_NAMED_GRAPH, String.class);
-        final Model model = createDefaultModel().read(in.getBody(InputStream.class), subject,
-                langFromMimeType(in.getHeader(Exchange.CONTENT_TYPE, String.class)));
+        final String namedGraph = in.getHeader(FCREPO_NAMED_GRAPH, "", String.class);
+        final Model model = createDefaultModel();
+        final String subject = getSubjectUri(exchange);
+
+        read(model, in.getBody(InputStream.class),
+                contentTypeToLang(parse(in.getHeader(CONTENT_TYPE, String.class)).getMimeType()));
```

Comment:
Does this change mean we're no longer limited to the types previously defined [here](https://github.com/fcrepo4-exts/fcrepo-camel/pull/125/files#diff-73dab38b27ed2d5973526f7469fb08e0L55)?